### PR TITLE
tyr: add test on instances api

### PR DESCRIPTION
**Pull request by kinnou02**
With this PR the most important API are tested
We need to add test on these remaining endpoint:
- billing_plan
- poi_type
- end_point
- job

### CODE REVIEW

File: source/tyr/readme.md [hunk 1]
```diff
@@ -79,6 +79,23 @@ with dev_settings.py the tyr settings file
 
 With those 2 files, running the commande ```honcho start``` in the tyr directory will launch all tyr services.
 
+##Running tests
+
+You need to install the dependancies for developements:
+```
+pip install -r requuirement_dev.txt
```
Comment (antoine-de) [inline source/tyr/readme.md]: typo on file name

File: source/tyr/tests/integration/instance_test.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    by default there is no traveler profile created for an instance
+    """
+    _, status = api_get('/v0/instances/fr/traveler_profiles/standard', check=False)
+    assert status == 404
+
+def test_create_empty_traveler_profile(create_instance):
+    """
+    we have created a profile with all default value, totally useless...
+    """
+    resp = api_post('/v0/instances/fr/traveler_profiles/standard')
```
Comment (antoine-de) [inline source/tyr/tests/integration/instance_test.py]: don't you want to test afterward that you can get it ?

File: source/tyr/tests/integration/instance_test.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+
+def test_create_traveler_profile(create_instance, traveler_profile_params):
+    resp = api_post('/v0/instances/fr/traveler_profiles/standard', data=json.dumps(traveler_profile_params),
+                    content_type='application/json')
+
+    for key, param in traveler_profile_params.iteritems():
+        assert resp[key] == param
```
Comment (antoine-de) [inline source/tyr/tests/integration/instance_test.py]: same here, maybe it nice to try to get it, and maybe add a test on one of it's param afterward

**Comment by antoine-de** [comment]
nice :+1:

File: source/tyr/migrations/versions/2e0ac7e0a7a7_add_traveler_profile_table.py [hunk 4]
```diff
@@ -50,3 +50,4 @@ def downgrade():
 
     #https://bitbucket.org/zzzeek/alembic/issues/159/opdrop_column-never-ends-with-an-enum
     sa.Enum(name='fallback_mode').drop(op.get_bind(), checkfirst=False)
+    sa.Enum(name='traveler_type').drop(op.get_bind(), checkfirst=False)
```
Comment (xlqian) [inline source/tyr/migrations/versions/2e0ac7e0a7a7_add_traveler_profile_table.py:53]: :+1:

**Comment by antoine-de** [comment]
Jenkins, test this please

**Pull request merged**